pkg/globalnet/controllers/gateway_monitor.go

```diff
@@ -506,6 +603,28 @@ func (g *gatewayMonitor) markRemoteClusterTraffic(remoteCidr string, addRules bo
 	}
 }
 
+func (l *LeaderElectionInfo) stop() {
+	if l.stopFunc != nil {
+		l.stopFunc()
+	}
+}
+
+func (l *LeaderElectionInfo) awaitStopped(timeout int) {
```

Comment:
The timeout would be clearer if it was passed directly as a `time.Duration` (and on `stopControllers`).